gensc/README.md

```diff
@@ -1,13 +1,19 @@
 ## Redirect for the Genomics Standards Consortium
 
-w3id.org/gensc
+The Genomic Standards Consortium currently has two w3id namespaces:
+- `gensc`
+- `mixs`
 
-This redirect is used to direct URIs for terms, checklists, packages and other documents related to GSC standards.
+The `mixs` namespace is more actively maintained, but some resources, like [Darwin Core](https://dwc.tdwg.org/) may need to temporarily use the `gensc` namespace, as pointed out by @MattBlissett in
+- https://github.com/perma-id/w3id.org/pull/3946
 
-The top-level redirects to the GSC website, which is currently house on a private server but will soon be hosted on github at https://github.com/GenomicsStandardsConsortium/gensc.github.io.
+Therefore, this `.htaccess` file redirects URLs in the form https://w3id.org/gensc/terms/MIXS:0000002 to https://w3id.org/mixs/0000002
```

Comment:
Similar to this, if we decide to add in the suggestions from https://github.com/perma-id/w3id.org/pull/4003#issuecomment-2030827886 we can add examples of what the redirections would look like.

Reply:
To me, the point of the `README.md` is to give **general orientation** to the contents of `.htaccess ` and other commentary that doesn't belong in `.htaccess `. Since the other two redirections under consideration have no replacement groups in them, I don't see a strong argument for reiterating them here.

Reply:
won't do because adds maintenance cost but little new knowledge